src/store/formDataSlice/formDataSlice.tsx

```diff
@@ -0,0 +1,53 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { PayloadAction } from '@reduxjs/toolkit';
+
+export interface IFormData {
+  name: string;
+  age: number;
+  email: string;
+  firstPassword: string;
+  secondPassword: string;
+  gender: string;
+  fileImage: string;
+  country: string;
+  termsAndConditions: boolean;
+}
+
+export const initialState = {
+  data: {
+    name: '',
+    age: 0,
+    email: '',
+    firstPassword: '',
+    secondPassword: '',
+    gender: '',
+    fileImage: '',
+    country: '',
+    termsAndConditions: false,
+  } as IFormData,
```

Comment:
better to create separate `type`  for `initialState`. try to avoid `as` as much as possible